pyproject.toml

```diff
@@ -15,15 +15,15 @@ backend-path = [
 name = "setuptools-scm"
 description = "the blessed package to manage your versions by scm tags"
 readme = "README.md"
-license.file = "LICENSE"
+license = "MIT"
+license-files = ["LICENSE"]
```

Comment:
I believe setuptools does not need an explicit `license-files` key according to [Controlling files in the distribution](https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html):
> * All files specified by the `license-files` configuration parameter in `pyproject.toml` and/or equivalent in `setup.cfg`/`setup.py`; note that if you don’t explicitly set this parameter, `setuptools` will include any files that match the following glob patterns: `LICEN[CS]E*`, `COPYING*`, `NOTICE*`, `AUTHORS**`;

@RonnyPfannschmidt Do you prefer implicit or explicit?